clang/lib/Format/Format.cpp

```diff
@@ -1893,6 +1938,8 @@ FormatStyle getChromiumStyle(FormatStyle::LanguageKind Language) {
   } else {
     ChromiumStyle.AllowAllParametersOfDeclarationOnNextLine = false;
     ChromiumStyle.AllowShortFunctionsOnASingleLine = FormatStyle::SFS_Inline;
+    ChromiumStyle.AllowShortFunctionsOnASingleLineOptions = {};
+    ChromiumStyle.AllowShortFunctionsOnASingleLineOptions.Inline = true;
```

Comment:
```suggestion
    ChromiumStyle.AllowShortFunctionsOnASingleLineOptions.Empty = true;
     ChromiumStyle.AllowShortFunctionsOnASingleLineOptions.Inline = true;
    ChromiumStyle.AllowShortFunctionsOnASingleLineOptions.Other = false;
```
`SFS_Inline` requires setting both `Empty` and `Inline`.

Reply:
Thanks, fixed!